apps/docs/app/src/routes/architecture/recovery-rotation/+page.md

```
@@ -1,270 +1,189 @@
+| `did`       | The identity's DID. Included in the signed payload — chains cannot be replayed across DIDs.   |
+| `seq`       | 1-based chain position. Strictly increasing, **no gaps**.                                     |
+| `prevRoot`  | Multibase key being retired. Statement 1's `prevRoot` MUST equal the genesis key.             |
+| `newRoot`   | Multibase key taking over as root.                                                            |
+| `rotatedAt` | ISO-8601 timestamp. Non-decreasing along the chain.                                           |
+| `signature` | Multibase Ed25519 signature by **`prevRoot`'s private key** over the canonical payload below. |
 
-### 3.3 Recovery verification rules
+**Canonical signing payload** — the RFC 8785 (JCS) serialization of:
 
-Resolvers and providers MUST:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)